rcl/src/rcl/arguments.c

```diff
@@ -235,6 +286,43 @@ rcl_parse_arguments(
       rcl_get_error_string());
     rcl_reset_error();
 
+    // Attempt to parse argument as log configuration file
+    if (RCL_RET_OK == _rcl_parse_external_log_config_file(argv[i], allocator, &args_impl->external_log_config_file)) {
+      continue;
+    }
+    RCUTILS_LOG_DEBUG_NAMED(ROS_PACKAGE_NAME,
+      "Couldn't parse arg %d (%s) as log config rule. Error: %s", i, argv[i],
+      rcl_get_error_string());
```

Comment:
I think this is out-of-date (I recommend updating your branches to make sure you're using master), it should be `rcl_get_error_string().str`, here and below.

Reply:
Added the `.str` to the end.